.github/workflows/cd.yml

```diff
@@ -1,18 +1,10 @@
 name: cd
 on:
-  workflow_run:
-    workflows: ["ci"]
-    branches-ignore: ["*"]
-    types:
-      - completed
   push:
     tags:
       - "v*"
 
-permissions:
-  contents: read
-
```

Comment:
Why is this no longer required?

Reply:
This is for our plugins that we control to avoid excess permissions . Now that we are going to create the releases from these pipelines this hardening is not required .